tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -304,7 +305,12 @@ def __gen_traffic(testbed_config,
             data_flow.tx_rx.port.tx_name = tx_port_name
             data_flow.tx_rx.port.rx_name = rx_port_name
 
-            eth, ipv4 = data_flow.packet.ethernet().ipv4()
+            eth, ipv4, udp = data_flow.packet.ethernet().ipv4().udp()
```

Comment:
Seems only packet-chassis will be affected by the backplane issue. Should we use an if..else.. to show it's platform specific behavior?